kotlin-mbedtls/src/main/kotlin/org/opencoap/ssl/transport/DtlsServer.kt

```diff
@@ -64,7 +64,15 @@ class DtlsServer(
             dtlsState is DtlsSession -> dtlsState.decrypt(buf)
 
             // no session, but dtls packet contains CID
-            cid != null -> ReceiveResult.CidSessionMissing(cid!!)
+            cid != null -> {
+                if (sslConfig.cidSupplier?.isValidCid(cid!!) == false) {
+                    logger.warn("[{}] Invalid CID {}", adr, cid!!.toHex())
```

Comment:
Lets use the convention from other log line:
```suggestion
                    logger.warn("[{}] [CID:{}] Invalid CID", adr, cid!!.toHex())
```